06-building-a-program-start-to-finish/solving-the-problem/Output/inverted-index.js

```diff
@@ -0,0 +1,128 @@
+
+/**
+ * @class InvertedIndex
+ */
+class InvertedIndexClass {
+  /**
+   * @constructor
+   */
+  constructor() {
+    this.files = {};
+    this.indexTable = {};
+  }
+  /**
+   * createIndex
+   * @param {Object} fileName name of file to map index for
+   * @return {Boolean} false if index is not created
+   */
+  createIndex(fileName) {
+    const currentFileContent = this.files[fileName];
+    this.indexTable[fileName] = this.indexTable[fileName] || {};
+    const fileLength = this.files[fileName].length;
+    for (let docIndex = 0; docIndex < fileLength; docIndex += 1) {
```

Comment:
I have looked into it thoroughly and I have discovered I can use forEach method  instead of using the 'for' loop. I have implemented the forEach method